skfem/utils.py

```diff
@@ -160,8 +153,7 @@ def build_pc_diag(A: spmatrix) -> spmatrix:
 
 
 def solver_iter_krylov(krylov: Optional[LinearSolver] = spl.cg,
-                       verbose: Optional[bool] = False,
-                       **kwargs) -> LinearSolver:
```

Comment:
I think we do still want to be able to parameterize Krylov solvers on definition; see #307.